ui/apps/platform/src/services/VulnerabilitiesService.js

```diff
@@ -35,19 +35,21 @@ function getBaseCveUrl(cveType) {
 /**
  * Send request to suppress CVEs with a given IDs.
  *
- * @param {!string} CVE unique identifier
- * @param {!string} CVE suppress duration, if 0 then CVEs are suppressed indefinitely
+ * @param {string} cveType The type of CVEs to suppress
+ * @param {string[]} cveNames CVE names to suppress
+ * @param {string} duration CVE suppress duration, in hours, if "0" then CVEs are suppressed indefinitely
  * @returns {Promise<AxiosResponse, Error>} fulfilled in case of success or rejected with an error
  */
-export function suppressVulns(cveType, cveNames, duration = 0) {
+export function suppressVulns(cveType, cveNames, duration = '0') {
```

Comment:
Even though it is marked optional here, `duration` is always provided with a value, even in VM 1.0 code. The `0` type is incorrect though, as the value passed through is always a string. `"0"`, `"24h"`, etc.